mido/__init__.py

```diff
@@ -92,23 +92,63 @@
     >>> get_input_names()
     ['MPK mini MIDI 1', 'SH-201']
 """
-import os
 
 from . import ports, sockets
 from .backends.backend import Backend
-from .messages import (Message, parse_string, parse_string_stream,
-                       format_as_string, MIN_PITCHWHEEL, MAX_PITCHWHEEL,
-                       MIN_SONGPOS, MAX_SONGPOS)
-from .midifiles import (MidiFile, MidiTrack, merge_tracks,
-                        MetaMessage, UnknownMetaMessage,
-                        bpm2tempo, tempo2bpm, tick2second, second2tick,
-                        KeySignatureError)
+from .messages import (
+    MAX_PITCHWHEEL,
+    MAX_SONGPOS,
+    MIN_PITCHWHEEL,
+    MIN_SONGPOS,
+    Message,
+    format_as_string,
+    parse_string,
+    parse_string_stream,
+)
+from .midifiles import (
+    KeySignatureError,
+    MetaMessage,
+    MidiFile,
+    MidiTrack,
+    UnknownMetaMessage,
+    bpm2tempo,
+    merge_tracks,
+    second2tick,
+    tempo2bpm,
+    tick2second,
+)
 from .parser import Parser, parse, parse_all
 from .syx import read_syx_file, write_syx_file
 from .version import version_info
 
-# Prevent splat import.
-__all__ = []
+__all__ = [
```

Comment:
Doing that will mean we'll have to `noqa` all of the imports, since they will seem unused to Ruff (and [mypy](https://mypy.readthedocs.io/en/stable/config_file.html#confval-implicit_reexport)).

Reply:
Wouldn't importing using full paths to the relevant module(s) mitigate this?
I think it's a better alternative to the "automagic" everything under `mido` that we have right now and is the direction I'm taking for mido 2.x.
Any further insights on the matter?

Reply:
Let's try with it then and I'll revert if it breaks something. Thanks.